hw1/src/main/resources/templates/movie_rating_list.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Movie Rating List</title>
+</head>
+<body>
+    <a href="/movies/new">Create a movie rating</a>
+    <br />
+    <table>
+        <tr>
+            <th>Movie</th>
+            <th>Rating</th>
+            <th>Name</th>
+        </tr>
+        <tr th:each="movieRating: ${movieRatings}">
+            <td th:text="${movieRating.title}"></td>
+            <td th:text="${movieRating.rating}"></td>
+            <td th:text="${movieRating.name}"></td>
```

Comment:
you captured the date but are not using it on any of your fields. The data is just in the background and not shown to the user.

Reply:
Added!